packages/store/src/IERC165.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.21;
 
-bytes4 constant ERC165_INTERFACE_ID = IERC165.supportsInterface.selector;
+bytes4 constant ERC165_INTERFACE_ID = type(IERC165).interfaceId;
```

Comment:
if the `interfaceId` is now built-in, do we need the constants anymore?

Reply:
Nope we don't need the constants! But I wanted to avoid the bigger diff of importing contracts in each touchpoint

Reply:
I think each place that uses this interface ID is also using the interface itself, so we should already be importing them?

Reply:
Implemented this and amending the commit. sorry for the messy history